src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx

```diff
@@ -13,33 +27,87 @@ class IntroductionPage extends React.Component {
   render() {
     return (
       <div className="schemaform-intro">
-        <FormTitle title="New 686" />
-        <p>qual to VA Form 21-686 (New 686).</p>
+        <FormTitle title="Add or remove dependents from your VA benefits" />
+        <p>
+          Equal to VA Form 21-686c (Application Request to Add And/Or Remove
+          Dependents) and/or Equal to VA Form 21-674 (Request for Approval of
+          School Attendance)
+        </p>
         <SaveInProgressIntro
+          {...this.props}
           hideUnauthedStartLink
+          verifiedPrefillAlert={VerifiedAlert}
           prefillEnabled={this.props.route.formConfig.prefillEnabled}
           messages={this.props.route.formConfig.savedFormMessages}
           pageList={this.props.route.pageList}
-          startText="Start the Application"
+          startText="Add or remove a dependent"
         >
           Please complete the 21-686 form to apply for declare or remove a
           dependent.
         </SaveInProgressIntro>
-        <h4>
-          Follow the steps below to apply for declare or remove a dependent.
-        </h4>
+        <h4>Follow the steps below to apply to add or remove a dependent.</h4>
```

Comment:
I chose not to update this to an `<h2>` as suggested in the ticket because the `<SaveInProgressIntro>` component uses an `<h3>`. There's currently a skip from `<h1>` in the `<FormTitle>` component to the aforementioned `<h3>`. That should be addressed at some point, but is outside the scope of this ticket for now.

Reply:
This was called out on our 508 by Jennifer S so we may need to circle back with her